src/15utility.js

```diff
@@ -571,7 +608,15 @@ var fileExists = utils.fileExists = function(path,cb){
                 cb(false);
             });
         });
-        //*/
+	} else if(utils.isReactNative) {
+        /* If ReactNative */

```

Comment:
As this is also stated in the `If` just above, I recommend ` /* If ReactNative */`gets removed